common/lib/xmodule/xmodule/js/spec_util.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    'use strict';
+
+    jasmine.waitForInputAjax = function (conditionalFn) {
```

Comment:
@andy-armstrong Actually this isn't just related to ajax, though most of the time the net effect is because of some ajax call somewhere. My plan was to make it more robust and place it somewhere in `js/libs` like a full library and include it with `require js` as shim. What do you think?